packages/compass-crud/src/stores/crud-store.ts

```diff
@@ -183,9 +195,20 @@ const extractCollectionStats = (collection: Collection): CollectionStats => {
     storage_size: coll.storage_size,
     free_storage_size: coll.free_storage_size,
     avg_document_size: coll.avg_document_size,
+    index_details: coll.index_details,
   };
 };
 
+function getDefaultSort(
+  collectionStats: CollectionStats | null
+): Sort | undefined {
+  if (collectionStats?.index_details._id_) {
+    return { _id: -1 };
```

Comment:
I'm now not sure. It seems like `{ $natural: -1 }` is actually just as fast as `{ $natural: 1 }` *(the default) at least in most cases. If it is in MOST cases we'd have to just change this logic and return `{ $natural: -1 }`. If it works and is fine in ALL cases, then most of this PR is probably not needed. We could just always use that as a default and we don't even need this function.

Reply:
But then.. I don't think we want $natural at all because it's just how documents are sorted in wiredtiger, which means any order really. For example, an update can bump an old document to the top of the natural order.